### PR TITLE
[Fix] Fix the issue with the height of big cta buttons on mobile devices

### LINKED ISSUE #35351: Text overflows out of button / button does not expand on homepage
**Describe the bug**
When accessing [freecodecamp.rocks](freecodecamp.rocks) via mobile, the 'Start coding(it's free)' button does not expand with respect to text.
As a result, the  text overflows the button area.

**To Reproduce**
1. Go to [freecodecamp.rocks](freecodecamp.rocks) via mobile
2. Scroll down to '....'
3. See error

**Expected behavior**
The button should expand appropriately and text should not overflow

**Screenshots**
screenshot_20190222-113920
screenshot_20190222-113933

**Smartphone (please complete the following information):**
 - OS: Android
 - Browser Chrome
 - Version 72.0.3626.105

### PR DESCRIPTION
**Pull request by Ensive**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #35351

### CODE REVIEW

**Comment by Ensive** [comment]
Tested locally through browserstack on the following devices:
Samsung Galaxy S7 (Chrome), iPhone X (Safari), iPhone 7 (Safari), Samsung Galaxy S4 (Chrome)

File: client/src/components/layouts/global.css [hunk 1]
```diff
@@ -1,10 +1,16 @@
 .btn-cta-big {
   max-height: 100%;
-  height: 70px;
+  height: auto;
   font-size: 40px;
   white-space: normal;
 }
 
+@media (min-width: 768px) {
```
Comment (Ensive) [inline client/src/components/layouts/global.css]: I have been thinking that starting from `769px` would make more sense, but I noticed that in the codebase we already have 768: https://github.com/freeCodeCamp/freeCodeCamp/blob/1f923db64b08d55deab4cca4be105efbe5b23163/client/src/templates/Challenges/components/video-modal.css#L5

Comment (rupali317) [reply client/src/components/layouts/global.css]: Is line 8 to 13 needed? If yes, why? because height: auto will do the trick in all device size

Comment (rupali317) [inline client/src/components/layouts/global.css]: 768px  is correct as tablet width range starts from 768px. You can go through this article http://www.drupalthemer.org/blog/responsive-design-media-queries-and-how-to-make-them-more-friendly

Comment (Ensive) [reply client/src/components/layouts/global.css]: @rupali317 
I've checked on production website, and indeed height is missing there for those buttons.
But, the `height: 70px` was added for some reason. I would like to hear from dev team whether or not they want to keep 70px height limit or we should be okay with `height: auto`.

Comment (Ensive) [reply client/src/components/layouts/global.css]: @rupali317 
Thanks for sharing. Good to be sure.

Comment (Bouncey) [reply client/src/components/layouts/global.css]: It was probably me that added the height rule, and css isn't my strong point.

If what you propose works, then I am fine about changing it. My code is not precious to me.

Comment (Ensive) [reply client/src/components/layouts/global.css]: `70px` makes sense to me for desktop devices.

Comment (rupali317) [reply client/src/components/layouts/global.css]: Even in desktop devices i doubt 70px will be correct as users can always resize the browser and we may see this issue again

Perhaps you can test only with height: auto and it should be fine. Let me know if that works

Comment (Ensive) [reply client/src/components/layouts/global.css]: @rupali317 
Let me check

Comment (ahmadabdolsaheb) [reply client/src/components/layouts/global.css]: height: auto fixes it on my side without the media queries.

**Review by ahmaxed — COMMENTED** [review]
height: auto without the media queries resolves the issue on my side.

File: client/src/components/layouts/global.css (hunk 1 above)
Comment (Ensive) [reply client/src/components/layouts/global.css]: Updated the PR

**Comment by Ensive** [comment]
@ahmadabdolsaheb 
@rupali317 
Since `height: auto` is applied by default. I removed `height: 70px` from selector.
Works for me locally. Please review.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**